reports/Florida.md

```diff
@@ -91,3 +93,70 @@ id: fl-tampa-1
 * https://twitter.com/jamiebullockk/status/1268658254806163460
 * https://twitter.com/InesseC/status/1268912900565094401
 * [Location on Google Street View](https://www.google.com/maps/@27.9563654,-82.4569108,3a,75y,255.48h,92.17t/data=!3m6!1e1!3m4!1sVIFVQFp3QMX517HfhvNfbg!2e0!7i16384!8i8192)
+* https://twitter.com/Psychedelicgigi/status/1268740374903894016
+* https://twitter.com/drewthehouse/status/1268713991230062593
+
+### Police fire tear gas at peaceful protest | June 4th
+
+Police fire tear gas at peaceful protestors, while simultaneously ordering protestors to lie down, forcing them to remain in the immediate area.
+
+tags: tear-gas, arrest
+
+id: fl-tampa-2
+
+**Links**
+* https://twitter.com/greg_doucette/status/1268694786485166081
+
+### Police pepper-spray police from protest | June 5th
+
+Supercut video from June 5th shows multiple incidents where police sprayed peaceful protestors, including protestors attempting to flee the scene. In addition, footage from the cut shows police shoving and pushing protestors before spraying
+
+tags: pepper-spray, spray
+
+id: fl-tampa-3
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1268765759783796736
+
+### Police kneel on protestor's neck to make arrest | June 5th
+
+Supercut video from June 5th shows police arresting a protestor while kneeling on the protestor's kneck. Other officers spray surrounding area with pepper spray.
+
+tags: pepper-spray, spray, arrest, knee, knee-on-neck
+
+id: fl-tampa-4
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1268765759783796736
```

Comment:
This incident should be removed because it is already listed above as `fl-tampa-1` (same victim in the purple shirt). Do combine any tags or further details this report adds to the existing entry though please.